monkestation/code/modules/a_ship_in_need_of_breaking/ships.dm

```diff
@@ -137,3 +137,11 @@
 	template_id = "wanglius"
 	description = "A product from DONGCORP, the Wanglius Class Salvage Tug is the tip of the spear when it comes to salvage vessels. Featuring a girthy crew quarters section, multiple engines for maximum thrust, engineering bay, sick bay, and a large and spacious cargo bay with a front access for releasing a load of hard working spacers on that pristine wreck. Guaranteed to satisfy."
 	mappath = "_maps/~monkestation/shipbreaking/wanglius.dmm"
+
+/*
+/datum/map_template/shipbreaker/whale
+	name = "Whale"
+	template_id = "whale"
+	description = "Whale-type description"
+	mappath = "_maps/~monkestation/shipbreaking/whale.dmm"
+*/
```

Comment:
This map template had no datum, so I made a quick and dirty one. However, as I'm unsure if this map template is wanted, I've commenting it out.